changelogs/unreleased/173-okuuva

```diff
@@ -0,0 +1,3 @@
+## All changes
```

Comment:
I think we can delete this change log file from PR.
The plugin's release note is generated manually, not like the Velero release process.

Reply:
Done. First tried to just revert the commit but forgot to add the signoff to the commit message 😓 So I just ended up dropping both the revert commit and the original with rebase.